ui/apps/platform/cypress/integration/compliance-enhanced/ComplianceEnhanced.helpers.js

```diff
@@ -5,9 +5,8 @@ import { visit } from '../../helpers/visit';
 
 export const basePath = '/main/compliance-enhanced';
 export const statusDashboardPath = `${basePath}/status`;
-export const clusterCompliancePath = `${basePath}/cluster-compliance`;
-export const clusterComplianceCoveragePath = `${clusterCompliancePath}/coverage`;
-export const clusterComplianceScanConfigsPath = `${clusterCompliancePath}/scan-configs`;
+export const clusterComplianceCoveragePath = `${basePath}/coverage`;
+export const clusterComplianceScanConfigsPath = `${basePath}/schedules`;
```

Comment:
What do you think about same names as in routePaths.ts file?
* `complianceEnhancedCoveragePath`
* `complianceEnhancedSchedulesPath`

We might also brainstorm pro and con of with or without `Enhanced` especially within tests folder.

Reply:
Good catch, those routes should be what you suggested. Yeah I'm still not a huge fan of the word `Enhanced` being thrown around everywhere in compliance v2. I guess having the keyword in the file name makes it clear enough as to what we're dealing with.